packages/gnome_terminal.rb

```diff
@@ -3,43 +3,29 @@
 class Gnome_terminal < Package
   description 'The GNOME Terminal Emulator'
   homepage 'https://wiki.gnome.org/Apps/Terminal'
-  @_ver = '3.38.2'
-  version @_ver
+  version '3.39.90'
```

Comment:
What not change the `@_ver` variable?

Reply:
If you'd like to keep it we can i guess

Reply:
The entire point of the ```@_ver``` variable is to capture the source package version, as opposed to the chromebrew package version. This way we can (hopefully)  just insert ```#{@_ver}``` into URLS and not have to update that every time we uprev from an upstream package update.